Could not find a declaration file for module 'debug'

I'm building a TS nodeJS project which uses:

* `typescript@4.5.4`
* `@mswjs/data@0.10.1`

But I have an error when I try to build the project:

```
node_modules/@mswjs/interceptors/lib/Interceptor.d.ts:1:26 - error TS7016: Could not find a declaration file for module 'debug'. '/projectFolder/node_modules/debug/src/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/debug` if it exists or add a new declaration (.d.ts) file containing `declare module 'debug';`

1 import { Debugger } from 'debug';
                           ~~~~~~~


Found 1 error.
```

Here is my `tsconfig.json` file:

```json
{
  "compilerOptions": {
    "module": "CommonJS",
    "target": "ES2017",
    "noImplicitAny": true,
    "preserveConstEnums": true,
    "outDir": "./dist",
    "sourceMap": true
  },
  "include": ["src/**/*"],
  "exclude": ["node_modules", "**/*.spec.ts"]
}
```

Does `@mswjs/interceptors` need to add `@types/debug` to its devDep and export the definitions or is there a misconfiguration on my side?

For now I fix the issue by `npm i -D @types/debug` but I don't think I should have to install the types in my consumer project since I do not directly use `debug` but `@mswjs/data` does.

Thanks for your help and for the great library 💪 .

--- a/package.json
+++ b/package.json
@@ -67,6 +67,7 @@
   },
   "dependencies": {
     "@open-draft/until": "^1.0.3",
+    "@types/debug": "^4.1.7",
     "@xmldom/xmldom": "^0.7.5",
     "debug": "^4.3.3",
     "headers-polyfill": "^3.0.4",

